backup/run_backup.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+echo "-- Database of Babel Backup and Rotation START --"
+
+timestamp=$(date +%Y_%m_%d_%H_%M_%S)
+backup_to_dropbox="python3 /root/backup/backup_to_dropbox.py"
+rotate_backup="python3 /root/backup/rotate_backups.py"
+backups_dir="/root/backup/backups"
+dump_file="dob_dump_$timestamp.sql"
+backup_file="$dump_file.gz"
+
+mkdir -p "${backups_dir}"
+cd "$backups_dir"
+
+if ! pg_dumpall -U $POSTGRES_USER --no-password >$dump_file; then
+	echo "pgdump failure!"
+	exit 1
+fi
+
+gzip $dump_file
+
+# Backup to Dropbox
+if ! $backup_to_dropbox $backup_file; then
+	echo "Dropbox backup failure!"
+	# Notify Slack
+	if [[ -n "$SLACK_INCIDENTS_WH_URL" ]]; then
+		curl -s -H 'content-type: application/json' \
+			-d "{ \"text\": \"🔴DoB Dropbox backup failure\nCould not upload \`${backup_file}\`.\" }" \
+			"$SLACK_INCIDENTS_WH_URL"
+	fi
+	exit 1
+fi
+
+# Dropbox backup rotation
+if ! $rotate_backup; then
+	echo "failed to rotate backups!"
+	# Notify Slack
+	if [[ -n "$SLACK_INCIDENTS_WH_URL" ]]; then
+		curl -s -H 'content-type: application/json' \
+			-d "{ \"text\": \"🔴DoB Dropbox backup rotation failure on ${timestamp}\" }" \
+			"$SLACK_INCIDENTS_WH_URL"
+	fi
+	exit 1
+fi
```

Comment:
I think this works for us right now, but shouldn't rotation should come before uploading the backup ? whatif the new backup has very high size and fails, i guess we should take this in consideration. Though shouldn't be an issue now.

Reply:
makes sense